setup-container.sh

```diff
@@ -46,7 +46,8 @@ GROUPNAME=$USER
 USERNAME=$USER
 EOF
 
-    j2 -o $tmpdir/Dockerfile $tmpdir/Dockerfile.j2 $tmpdir/data.env
+    j2 -ozzz $tmpdir/Dockerfile $tmpdir/Dockerfile.j2 $tmpdir/data.env || \
+    j2 $tmpdir/Dockerfile.j2 $tmpdir/data.env > $tmpdir/Dockerfile
```

Comment:
what is the purpose of this change?

Reply:
Same concern here. Tried `j2 -ozzz $tmpdir/Dockerfile $tmpdir/Dockerfile.j2 $tmpdir/data.env`, it failed with error like below:
```
$ j2 -ozzz $tmpdir/Dockerfile $tmpdir/Dockerfile.j2 $tmpdir/data.env
usage: j2 [-h] [-v] [-f {?,json,ini,env}] [-e VAR]
          [--filters python-file [python-file ...]]
          [--tests python-file [python-file ...]] [--customize python-file.py]
          [--undefined] [-o outfile]
          template [data]
j2: error: unrecognized arguments: /tmp/tmp.3LbDNSU8Uh/data.env
```

Reply:
My apologies. It is not needed. Removed